File: UI/app/src/main/java/com/rikkei/training/ui/base/BaseDialogFragment.java
package com.rikkei.training.ui.base;

import android.content.Context;
import android.view.LayoutInflater;

import androidx.fragment.app.DialogFragment;

public class BaseDialogFragment extends DialogFragment {
    protected LayoutInflater inflater;

    public BaseDialogFragment(Context context) {
        inflater = LayoutInflater.from(context);
    }
}

File: UIKotlin/app/src/main/java/com/rikkei/training/uikotlin/base/BaseDialogFragment.kt
package com.rikkei.training.uikotlin.base

import android.content.Context
import android.view.LayoutInflater

import androidx.fragment.app.DialogFragment

abstract class BaseDialogFragment(context: Context) : DialogFragment() {
    protected var inflater: LayoutInflater = LayoutInflater.from(context)

}
